app/collections/show-list.cjsx

```diff
@@ -140,14 +198,37 @@ module.exports = React.createClass
           meta = @state.subjects[0].getMeta()
 
           <div>
-            <div className="collection-subjects-list">
+          {if @state.selecting
+            <div className="collection-buttons-container">
+              <button
+                className="select-images-button"
+                onClick={@openCollectionsManager}
+                disabled={@state.selected.length < 1}>
+                <span>Add to Collection</span>
+              </button>
+              {if @state.collectionsManaging
+                <Dialog tag="div" closeButton={true} onCancel={@closeCollectionsManager}>
+                <CollectionsManager user={@props.user} project={@props.project} subjectIDs={@state.selected} onSuccess={@closeCollectionsManager} />
+                </Dialog>}
+              <button className="select-images-button" onClick={@deleteSubjects} disabled={@state.selected.length < 1}>Remove from Collection</button>
```

Comment:
The remove from collection button doesn't make sense if I'm viewing someone else's collection. It probably should only be available if I have appropriate permissions to edit that collection. I was able to use it and it looked like I _could_ and _did_ delete some subjects from a collection that I do not have permissions for. Refreshing the page showed they weren't actually deleted.